src/sentry/static/sentry/app/views/projectEnvironments.jsx

```diff
@@ -43,11 +156,17 @@ const ProjectEnvironments = createReactClass({
               <ListLink to={`/${orgId}/${projectId}/settings/environments/`} index={true}>
                 {t('Environments')}
               </ListLink>
+              <ListLink
+                to={`/${orgId}/${projectId}/settings/environments/hidden/`}
+                index={true}
+              >
+                {t('Hidden')}
+              </ListLink>
             </ul>
           }
         />
         <Panel>
-          <PanelHeader>{t('Environments')}</PanelHeader>
+          <PanelHeader>{this.state.title}</PanelHeader>
```

Comment:
might as well take this out of state on line 143, but also i think it's better to do your ternary to decide title in the render function and not have this on state at all

Reply:
(one benefit is that if isHidden is updated, title will always match it no matter what)

Reply:
Good point 👍